### PR TITLE
DRILL-6895: Warnings for license headers checks and wrong license hea…

### PR DESCRIPTION
**Pull request by vdiravka**
…ders in some files

To verify it please run:
`mvn license:check -Dlicense.skip=false`

See details in [DRILL-6895](https://issues.apache.org/jira/browse/DRILL-6895)

### CODE REVIEW

File: distribution/src/resources/drill-conf [hunk 1]
```diff
@@ -1,19 +1,21 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (arina-ielchiieva) [inline distribution/src/resources/drill-conf]: Not sure if this change is necessary. Please check here and below.

Comment (vdiravka) [reply distribution/src/resources/drill-conf]: It is good for python projects and for FreeBSD and OpenBSD systems.
None of these points satisfy Drill project. Therefore I will revert that changes and change `/usr/bin/env bash` in all other files.

Comment (vdiravka) [reply distribution/src/resources/drill-conf]: Done

**Comment by arina-ielchiieva** [comment]
+1

**Pull request merged**